lib/util/patterns/L.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
I can rename if you like, sure. The name was chosen (in JSCS) to match the shorthand POSIX character class `\p{L}`.

Can you give me some guidance on noting the license correctly? Thanks!